core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -788,6 +788,7 @@ private void handleLocaleCheck() {
     tableDrawerAdapter.setTableClickListener(null);
     tableDrawerAdapter = null;
     unbinder.unbind();
+    webViewList.clear();
```

Comment:
In my previous comment I was referring to both lines of change in this file. Is this needed?

Reply:
Yes, this one I am sure fixed a memory leak.

Reply:
cool